sdk/storage/Azure.Storage.DataMovement.Blobs/README.md

```diff
@@ -151,17 +151,59 @@ await tranfer.WaitForCompletionAsync();
 
 ### Initializing Blob Storage `StorageResource`
 
-Azure.Storage.DataMovement.Blobs exposes a `StorageResource` for each type of blob (block, page, append) as well as a blob container. Storage resources are initialized with the appropriate client object from Azure.Storage.Blobs.
+Azure.Storage.DataMovement.Blobs exposes `BlobsStorageResourceProvider` to create `StorageResource` instances for each type of blob (block, page, append) as well as a blob container. The resource provider should be initialized with a credential to properly authenticate the storage resources. The following demonstrates this using an `Azure.Core` token credential.
+
+```C# Snippet:MakeProvider_TokenCredential
+BlobsStorageResourceProvider blobs = new(tokenCredential);
+```
+
+For workflows that require specific credentials for specific Blob Storage resources, the resource provider can be initialized with a delegate to construct the appropriate credential from the resource's URI. The following demonstrates a provider that produces a sas scoped to the resource from a `StorageSharedKeyCredential`.
+
+```C# Snippet:MakeProvider_SasFactory
```

Comment:
I wonder how prominent we want to make this? I feel like using this delegate will be a more advanced scenario and I wouldn't want customers to just default to doing that. We have the token cred sample above but since it's just one line, it's easy to miss (I missed it at first and thought this was our first auth sample). Not exactly sure how to make it less prominent while still having the info but just a thought.

(This goes for Files as well)

Reply:
Hmm it does feel like we really need to have some sort of sample like this, as SAS workflows can get complex fast and customers need to know how to do something like this. But we can probably move this elsewhere and encourage users getting started to have the sas in the uri rather than through the SasCredential type.

Reply:
We can leave this in the cs files but remove them from the readmes for now while we figure out the direction we want to guide people